audit/example_test.go

```diff
@@ -1,25 +0,0 @@
-package audit_test
```

Comment:
the same to me: it is either we have it here and with examples, or `audit` is moved into the IR app's `internal` fully

Reply:
Moving to IR can be considered, clients can do nothing with this.

Reply:
i always commend SDK trimming, so lets completely replace the package

Reply:
purged